src/main/java/com/yahoo/bullet/querying/Querier.java

```diff
@@ -298,8 +298,7 @@
     @Getter(AccessLevel.PACKAGE) @Setter(AccessLevel.PACKAGE)
     private Scheme window;
 
-    // For testing convenience
-    @Getter(AccessLevel.PACKAGE)
+    @Getter
```

Comment:
You can change back to package level.

Reply:
It's a bit weird to expose the Query but not the RunningQuery if you have need for the start time etc. Might as well.